### PR TITLE
Add apt-get instructions in README

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,16 @@ This is part of the [Mono project](http://www.mono-project.com/).
 
 This requires the libraries used by the Cairo vector graphics library to build (freetype2, fontconfig, Xft2 and libpng).
 
-On OSX you can use [Homebrew](https://brew.sh/) to install the dependencies:
+On **OSX** you can use [Homebrew](https://brew.sh/) to install the dependencies:
 
 	brew install glib cairo libexif libjpeg giflib libtiff autoconf libtool automake
 
-On Windows you can use [Chocolatey](https://chocolatey.org) to install the dependencies. Run the following commands from the root of the repository from an admin command prompt:
+On **Linux** you can use `apt-get` to install the dependencies:
```
Comment (akoeplinger) [inline README.md]: apt-get is only used on Debian-based distros so probably better to be explicit:

`On Debian-based Linux distributions you can use apt-get to install the dependencies`

File: README.md [hunk 2]
```diff
@@ -6,11 +6,16 @@ This is part of the [Mono project](http://www.mono-project.com/).
...
+
+	sudo apt-get update
```
Comment (akoeplinger) [inline README.md]: I wouldn't mention this, we can assume the apt cache is up to date on a normal developer machine.

File: README.md [hunk 3]
```diff
@@ -6,11 +6,16 @@ This is part of the [Mono project](http://www.mono-project.com/).
...
+	sudo apt-get install libgif-dev xvfb autoconf libtool automake build-essential gettext cmake git libglib2.0-dev libcairo2-dev libtiff-dev libexif-dev
```
Comment (akoeplinger) [inline README.md]: xvfb shouldn't be needed, I guess this just brought in some necessary dependency?

Same for cmake and git.

Comment (hughbe) [reply README.md]: Only included these as they were included in the `.travis.yml` file:
https://github.com/mono/libgdiplus/blob/master/.travis.yml

File: README.md (hunk 2 above)
Comment (hughbe) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (hughbe) [reply README.md]: Right - fixed

File: README.md (hunk 3 above)
Comment (akoeplinger) [reply README.md]: Ah. Yeah those are just there so we can compile Mono and run the System.Drawing+WinForms tests :)

**Pull request merged**